frontend/vite.config.ts

```diff
@@ -30,5 +31,40 @@ export default defineConfig(() => {
         '@public': path.resolve(__dirname, './public'),
       },
     },
+    test: {
+      globals: true,
+      environment: 'jsdom',
+      setupFiles: ['./src/vitest-setup.ts'],
+      // For this config, check https://github.com/vitest-dev/vitest/issues/740
```

Comment:
Not sure if I get this?

```suggestion
      // To understand this config, check https://github.com/vitest-dev/vitest/issues/740
```